.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
-  - node
+  - 8
   - 6
-  - 4
```

Comment:
Brunch supports Node 4+. https://github.com/brunch/brunch/blob/master/package.json#L47

Maybe safer to keep node:4 container to test.